tools/releaseBuild/Images/microsoft_powershell_windowsservercore/DockerFile

```diff
@@ -1,6 +1,6 @@
 # escape=`
 #0.3.6 (no powershell 6)
-FROM microsoft/powershell:windowsservercore
+FROM mcr.microsoft.com/powershell:windowsservercore
```

Comment:
I noticed servercore dockerfile wasn't updated in the last PR (and it is using strange FROM location). Just FYI.

Reply:
That is just a sample in PSRelease repo.